File: src/main/java/com/example/eyeofsauron/controller/MockAuthController.java
package com.example.eyeofsauron.controller;

import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Map;

import static java.util.Collections.singletonMap;

@RestController
public class MockAuthController {
    @PostMapping("/api/rest-auth/login")
    public Map<String, String> auth() { return singletonMap("key", "12345678980"); }
}

File: src/main/kotlin/com/example/eyeofsauron/controller/MockAuthController.kt
package com.example.eyeofsauron.controller

import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RestController

/**
 * Временная заглушка для системы авторизации фронтального приложения
 */
@RestController
class MockAuthController {
    @PostMapping("/api/rest-auth/login")
    fun auth() = mapOf("key" to "12345678980")
}